manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: sxyazi.yazi
+PackageVersion: 0.2.5
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: yazi-x86_64-pc-windows-msvc\ya.exe
+  PortableCommandAlias: ya
+- RelativeFilePath: yazi-x86_64-pc-windows-msvc\yazi.exe
+  PortableCommandAlias: yazi
+Installers:
+- Architecture: x64

```

Comment:
Can we also add [Windows aarch64](https://github.com/sxyazi/yazi/releases/download/v0.2.5/yazi-aarch64-pc-windows-msvc.zip) here?

Reply:
Done :)